a3p-integration/proposals/a:upgrade-next/package.json

```diff
@@ -6,6 +6,7 @@
     "upgradeInfo": {
       "coreProposals": []
     },
+    "comment": "all the priceFeed submissions go in the same directory",
```

Comment:
we've been using `$comment` to make clear it's not part of the data
https://github.com/Agoric/agoric-sdk/blob/b20a8b681bb194e22036d57f4ee6eb2a8c8aa549/packages/vm-config/decentral-core-config.json#L2

however I'm not sure what this comment serves. What's the misunderstanding it's meant to remedy?